packages/workflows/src/definition/inventory/create-inventory-item.ts

```diff
@@ -0,0 +1,81 @@
+import { InputAlias, Workflows } from "../../definitions"
+import {
+  TransactionStepsDefinition,
+  WorkflowManager,
+} from "@medusajs/orchestration"
+import { exportWorkflow, pipe } from "../../helper"
+
+import { InventoryTypes, WorkflowTypes } from "@medusajs/types"
+import { InventoryHandlers } from "../../handlers"
+
+export enum CreateInventoryItemActions {
+  prepare = "prepare",
+  createInventoryItems = "createInventoryItems",
+}
+
+const workflowSteps: TransactionStepsDefinition = {
+  next: {
+    action: CreateInventoryItemActions.prepare,
+    noCompensation: true,
+    next: {
+      action: CreateInventoryItemActions.createInventoryItems,
+    },
+  },
+}
+
+const handlers = new Map([
+  [
+    CreateInventoryItemActions.prepare,
```

Comment:
todo: the prepare does not have to be here, it can be pass to the exportWorkflow as the third argument